src/create-theme-provider.js

```diff
@@ -0,0 +1,92 @@
+const React = require('react');
+const PropTypes = require('prop-types');
+const isFunction = require('is-function');
```

Comment:
You don't need this package, simple `typeof === 'function'` will work here, because we don't need to workaround old browsers or iframes.